### DISCUSSION TITLE
一分钟读论文：《战争与和平：世界政治对软件生态系统的影响》

### DISCUSSION THREAD

**Discussion by giscus** [unbug/unbug.github.io#26] [Announcements]
# 一分钟读论文：《战争与和平：世界政治对软件生态系统的影响》

两国交战，平民是否有罪？国际人道法约定“战争时要尽可能限制对妇女和儿童以及其他平民的影响”。开源许可证明确规定应该“不歧视个⼈或群体。License 不得歧视任何⼈”。 俄乌冲突中开源如何影响开发者呢？我们来看看⽇本学技术研究所和澳大利亚墨尔本大学合著的论文《In war and peace: the impact of world politics on software ecosystems》中研究的几个例子：当开源项目的维护者将他们的开源项目化为抗议软件（Protestware）。

https://unbug.github.io/In-war-and-peace-the-impact-of-world-politics-on-software-ecosystems/

**Comment by unbug** [comment]
- 当战争时，License 就是一张废纸
- License 似乎只限制使用者，作者的职责是一片空白
